papas/pfobjects.py

```diff
@@ -198,12 +208,23 @@ def __init__(self, mother, *args, **kwargs):
 class MergedCluster(Cluster):
     '''The MergedCluster is used to hold a cluster that has been merged from other clusters '''
 
-    def __init__(self, mother):
-        self.mother = mother
-
+    def __init__(self, clusters, identifiervalue = None):
```

Comment:
We want a mergedcluster to have a unique identifier that allows the mergedclusters to be sorted on total energy/pt. So we need to know the total energy of the merged cluster at the time we create it. In my first attempt I totalled the energy and then passed this to the constructor along with the first cluster and then added in the other clusters one by one which works but is not very nice. In your first look at this pull request you were not too happy with this. A cleaner approach is to pass a list of clusters to the constructor, then we can determine the total energy in order to create the unique identifier. This is what I implemented.